agent/http_test.go

```diff
@@ -264,6 +264,22 @@ func TestSetKnownLeader(t *testing.T) {
 	}
 }
 
+func TestSetFilteredByACLs(t *testing.T) {
+	t.Parallel()
+	resp := httptest.NewRecorder()
+	setResultsFilteredByACLs(resp, true)
+	header := resp.Header().Get("X-Consul-Results-Filtered-By-ACLs")
```

Comment:
nit picking but how about `X-Consul-Count-Filtered-By-ACLs` / `X-Consul-Count-Results-Filtered-By-ACLs` ? I know it's a bit on the long side

and should it be `ACL` ?